### DISCUSSION TITLE
Official GitHub account on Mastodon

### DISCUSSION THREAD

**Discussion by rhysd** [community/community#44940] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Product Feedback

### Body

Would you provide [Mastodon](https://joinmastodon.org/) account from GitHub officially? I'd like to subscribe GitHub's updates there. I'd expect same posts as tweets on the official Twitter account.

https://twitter.com/github

**Comment by Groxx** [comment]
This is extra relevant now that Twitter has cut off public viewing.  It's sorta no longer reasonable to use it for site-status and the like.
